def/babel-core.ts

```diff
@@ -2,6 +2,7 @@ import { Fork } from "../types";
 import typesPlugin from "../lib/types";
 import sharedPlugin from "../lib/shared";
 import es7Def from "./es7";
+import * as N from "../gen/nodes";
```

Comment:
> Do you think it's okay to refer to generated code in these def/*.ts files? Is there a better way to do this?

Was trying to avoid consuming any of the gen exports outside of `/gen` and `main.ts` because the type is dependent on which defs are loaded. Like for `NumericLiteral` below, suppose some other def adds some fields on it. Since gen is based on the defs used in main, if someone uses fork and doesn't provide the same defs, the type will be wrong.

Bummer.

Some Options:
1) Say the case I explained above would be rare and just use gen

2) Run the gen on a def-by-def basis and import that type here. And, separately keep running the overall gen to be exported by main.

3) Define an interface here to be used in `getDefault` and extended by the gen node type somehow. Though it would be weird for some types to have interfaces and others not. And annoying to keep the interface and def in sync.

4) Use `any`. No type safety. :/

5) Use an inline type for the fields used in `getDefault`. No real type safety since we could say the type is anything - there's nothing checking that the inline interface actually matches the fields.

---

Option 2 doesn't seem so bad. And makes it so the only fields available are the fields defined in the same def. So we don't get any accidental dependencies added by other defs.